src/components/TodosList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class TodosList extends React.Component {
+  render() {
+    const {
+      todos, handleChangeProps, handleDelete, setUpdate,
+    } = this.props;
+    return (
+      <ul>
+        {todos.map((todo) => (
+          <TodoItem
+            key={todo.id}
+            todo={todo}
+            handleChangeProps={handleChangeProps}
+            handleDelete={handleDelete}
+            setUpdate={setUpdate}
+          />
+        ))}
+      </ul>
+    );
+  }
+}
+
+TodosList.propTypes = {
+  todos: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      title: PropTypes.string.isRequired,
+      completed: PropTypes.bool.isRequired,
+    }),
+  ).isRequired,
+  handleChangeProps: PropTypes.func.isRequired,
+  handleDelete: PropTypes.func.isRequired,
+  setUpdate: PropTypes.func.isRequired,
+};
```

Comment:
You did an excellent job on checking your Prop Types! :+1: :100:

Reply:
:grin: maybe it would be nice to have these PropTypes defined in a single place of the project? Particularly the ones representing a todo.